float32 Vx
float32 Vy
float32 Vz
float32 X
float32 Y
float32 Z
float32 yaw
float32 Aim_X
float32 Aim_Y
float32 Aim_Z
float32 Vx_now
float32 Vy_now
float32 Vz_now

